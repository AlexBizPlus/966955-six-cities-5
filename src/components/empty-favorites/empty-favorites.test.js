import React from "react";
import renderer from "react-test-renderer";
import EmptyFavorites from "./empty-favorites";

describe(`Render EmptyFavorites`, () => {
  test(`Render EmptyFavorites`, () => {
    const tree = renderer
      .create(
          <EmptyFavorites />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
