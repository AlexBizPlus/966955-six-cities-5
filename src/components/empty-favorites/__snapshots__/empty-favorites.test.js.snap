// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render EmptyFavorites Render EmptyFavorites 1`] = `
<section
  className="favorites"
>
  <h1
    className="visually-hidden"
  >
    Favorites (empty)
  </h1>
  <div
    className="favorites__status-wrapper"
  >
    <b
      className="favorites__status"
    >
      Nothing yet saved.
    </b>
    <p
      className="favorites__status-description"
    >
      Save properties to narrow down search or plan yor future trips.
    </p>
  </div>
</section>
`;
